src/net/sf/memoranda/ui/AppFrame.java

```diff
@@ -689,8 +689,7 @@ protected void processWindowEvent(WindowEvent e) {
         }
         else if ((e.getID() == WindowEvent.WINDOW_ICONIFIED)) {
             super.processWindowEvent(new WindowEvent(this,
-                    WindowEvent.WINDOW_CLOSING));
-            doMinimize();
+                    WindowEvent.WINDOW_ICONIFIED));
```

Comment:
One theory I came up with is that maybe they intended to have the window minimize button not minimize to the taskbar but instead add an icon to the system tray, which could then be clicked to re-create the window.  The current version of the program doesn't add any icons to the system tray, so perhaps they just never got around to implementing it.

Of course, I'm thinking in terms of the Windows OS - I'm not familiar enough with Mac or Linux to make any comments on those.

What do you think? Does that theory hold water?